sbgECom library being linked before declared

If you are generating the project on Windows, the CMakeList in the libraries folder tries to use the sgbECom library on [line 10](https://github.com/uorocketry/rocket-code-2020/blob/aa797b100dc746e02cd8536a9654c11ec979f2cf/libraries/CMakeLists.txt#L10) before it is declared in the same file on [line 22](https://github.com/uorocketry/rocket-code-2020/blob/aa797b100dc746e02cd8536a9654c11ec979f2cf/libraries/CMakeLists.txt#L22).


```console
CMake Error at libraries/CMakeLists.txt:10 (target_link_libraries):
  Cannot specify link libraries for target "sbgECom" which is not built by
  this project.
```

I managed to fix this by moving the `target_link_libraries` to right bellow `add_library`.
```cmake
...
add_library(sbgECom STATIC ${SRC} ${COMMON_SRC} ${SM_SRC})

if (MSVC)
    target_link_libraries(sbgECom Ws2_32)
endif()
...
```

--- a/libraries/CMakeLists.txt
+++ b/libraries/CMakeLists.txt
@@ -6,8 +6,6 @@ file(GLOB_RECURSE SRC ${PROJECT_SOURCE_DIR}/libraries/sbgECom/src/*.c)
 file(GLOB_RECURSE COMMON_SRC ${PROJECT_SOURCE_DIR}/libraries/sbgECom/common/*.c)
 file(GLOB_RECURSE SM_SRC ${PROJECT_SOURCE_DIR}/libraries/sbgECom/src/*.cpp)
 
-
-
 #
 #	Exclude platform specific files
 #
@@ -24,7 +22,6 @@ if (MSVC)
 endif()
 
 target_include_directories(sbgECom PUBLIC ./sbgECom/common ./sbgECom/src)
-
 
 # ---------------
 # WiringPi library

